en/admin-guide/datamodel-tool/index.rst

```diff
@@ -0,0 +1,74 @@
+Data-model tool
+===============
+
+The plugin contains a data-model tool capable of initializing and upgrading a QGEP database.
+
+It is aimed at system administrators and power users, and should not be used by end users, as it
+could lead to data loss if not used properly.
+
+For advanced cases, it is also possible to `upgrade your database manually <../db-update-pum/index.html>`_
+
+Setup
+-----
+
+The data-model tool is hidden by default. To make it visible, you must first enable ``Admin mode`` 
+in the plugin settings :
+
+* Go to ``Plugin > QGEP > Settings``
+
+* Switch to the ``Developer options``
+
+* Check ``Admin mode``
+
+* Restart QGIS
+
+The data-model tool should now appear under ``Plugin > QGEP > Datamodel``
+
+Usage
+-----
+
+Main dialog
+^^^^^^^^^^^
+
+The main dialog looks like this.
+
+.. figure:: images/datamodel-tool.png
+
+* **Release**:
+
+    * **Version**: the version to retrieve the files from. Unless ``developper mode`` is enabled in the QGEP settings, only the latest stable version is available. Selecting anything different from the latest stable version should only ever be used on a testing database, as it could upgrade the database to an inconsistent state.
+
+* **Python dependencies**:
+    
+    * **Install**: If there are missing requirements, click this to try to install them automatically. Note that this will not work on all systems. If it fails, you can try to start QGIS with administrator privileges, or `install the dependencies manually <../db-update-pum/index.html#requirements>`_.
+
+* **Postgres service configurations**:
+
+    * **File location**: shows the location of the ``pg_service.conf`` file being used
+
+    * **PG Config**: allows to select the postgres service configuration of the database on which you want to operate.
```

Comment:
From my experience it does not select automatically the database from the project that is open - it displays the first pg_config entry - that is a little bit dangerous from a users perspectiv and should may be highlighted or a selection should actively be inforced and not just the first in the list be taken.

Reply:
Ok, will add a warning in the docs. Stefan created an issue for that already : https://github.com/QGEP/QGEP/issues/712